README.md

```diff
@@ -103,3 +100,61 @@ $ ./setup <app name> <plan>
 $ heroku kafka:topics:write [prefix]textlines "hello world" -a <app>
 $ heroku pg:psql -c 'select * from windowed_counts' HEROKU_POSTGRESQL_URL -a <app>
 ```
+
+### Example Use Cases
+
+Now let's use Kafka Streams with some example use cases. The `data-generators` directory contains some simple Ruby scripts to generate streams of data. Instructions on how to use them are below.
+
+#### Word Count
+
+First we'll do word count over a large stream of text. This will produce into Kafka lines from _Alice's Adventures in Wonderland_.
+
+```bash
+$ heroku run ruby data-generators/text-generator/stream-lines-to-kafka.rb data-generators/text-generator/alice-in-wonderland.txt
```

Comment:
Will running in a one-off dyno count towards the free dyno count?

Reply:
Looks like it doesn't.

However, we should probably add `--app sushi` like other statements.

Reply:
Added `--app sushi` anywhere I use the Heroku CLI